Changes

```diff
@@ -14,6 +14,9 @@ LIST OF CHANGES
  - Extended the code for the samplesheet daemon to generate samplesheets for
    NovaSeqX instruments.
  - Removed remaining unused test data from the era of xml st::api::lims driver.
+ - update signature and iRODS documentation link in template for emails
+ - remove link to illumina/runs qc page in emails reporting status when run
+   reaches qc complete
```

Comment:
The emails are triggered by a number of statuses, not only by 'qc complete'. That's why the text say 'this run will be/is available'